odoo/tests/common.py

```diff
@@ -1728,7 +1728,7 @@ def authenticate(self, user, password, browser: ChromeBrowser = None):
 
         return session
 
-    def browser_js(self, url_path, code, ready='', login=None, timeout=60, cookies=None, error_checker=None, watch=False, **kw):
+    def browser_js(self, url_path, code, ready='', login=None, timeout=60000, cookies=None, error_checker=None, watch=False, **kw):
```

Comment:
Could you check if you left something like this in your numpad pr?

Reply:
Yes :) i checked. It's fine